tests/unit_tests/testdaemonaccesscontrol.cpp

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "testdaemonaccesscontrol.h"
+
+#include "daemon/daemonaccesscontrol.h"
+
+void TestDaemonAccessControl::initTestCase() {
+  // Reset the session before each test case.
+  DaemonAccessControl::instance()->resetSession();
+}
+
+void TestDaemonAccessControl::testInactiveCommands() {
+  auto dac = DaemonAccessControl::instance();
+
+  QVERIFY(dac->isCommandAuthorizedForPeer("status", nullptr));
+  QVERIFY(!dac->isCommandAuthorizedForPeer("logs", nullptr));
+  QVERIFY(!dac->isCommandAuthorizedForPeer("cleanlogs", nullptr));
+  QVERIFY(!dac->isCommandAuthorizedForPeer("deactivate", nullptr));
+
+  // A peer id of -1 means activation failed,
+  // access crontrol defaults to unauthorized.
+  dac->setPeerId(-1);
+  QVERIFY(!dac->isCommandAuthorizedForPeer("activate", nullptr));
+  // Set peer id to something valid for activateion to succed.
+  dac->setPeerId(1);
+  QVERIFY(dac->isCommandAuthorizedForPeer("activate", nullptr));
```

Comment:
Consider adding a couple of lines to verify that "status" is authorized for any peer when the VPN is inactive.